src/wallets/coinbase/actions.ts

```diff
@@ -0,0 +1,232 @@
+import { ElementHandle, Page } from 'playwright-core';
+import { AddNetwork, AddToken } from '../..';
+import { waitForChromeState } from '../../helpers';
+import { performPopupAction } from '../metamask/actions';
+import { WalletOptions } from '../wallets';
+import { extensionUrl } from './coinbase';
+
+const goHome = async (page: Page): Promise<void> => {
+  await page.getByTestId('portfolio-navigation-link').click();
+};
+
+export async function getStarted(
+  page: Page,
+  {
+    seed = 'already turtle birth enroll since owner keep patch skirt drift any dinner',
+    password = 'password1234!!!!',
+  }: WalletOptions,
+): Promise<void> {
+  // Welcome screen
+  await page.getByTestId('btn-import-existing-wallet').click();
+
+  // Import Wallet
+  await page.getByTestId('btn-import-recovery-phrase').click();
+  await page.getByTestId('seed-phrase-input').fill(seed);
+  await page.getByTestId('btn-import-wallet').click();
+  await page.getByTestId('setPassword').fill(password);
```

Comment:
some of these test-ids are camel-case vs kebab-case, is that just a coinbase wallet thing?

Reply:
yea, that's not on us.